File: core-java-practice/gcr-codebase/programming-elements/powerCalculation.java
import java.util.*;
public class powerCalculation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("base: ");
        int base = scanner.nextInt();
        System.out.print("exponent: ");
        int exponent = scanner.nextInt();
        double result = Math.pow(base, exponent);
        System.out.println("the result of " + base + " raised to the power of " + exponent + " is:" + result);
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/PowerCalculation.cs
using System;

class PowerCalculation
{
    static void Main()
    {
        double baseNumber = 2.0;
        double exponent = 3.0;
        double result = Math.Pow(baseNumber, exponent);
        
        Console.WriteLine($"{baseNumber} raised to the power of {exponent} is {result}");
    }
}